CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing 🙌
```

Comment:
Having emojis is not the style we follow in other docs.

Reply:
Understood. Will remove.

Reply:
Resolved via https://github.com/JetBrains/compose-jb/pull/511/commits/d64043f4cc247009deb3e6e8f7c35a8f7bfba622